scripts/insert_into_appointments.sql

```diff
@@ -1 +1,8 @@
 -- insert appointments creating connections between doctors and patients
+INSERT INTO appointments (patient_id, doctor_id, time, date)
+    VALUES(8, 4, '10am', 'August 12, 2016'),
+          (5, 6, '1pm', 'August 17, 2016'),
+          (8, 6, '10am', 'August 12, 2016'),
+          (7, 4, '1pm', 'August 17, 2016')
```

Comment:
Hard-coding values like this will work, so I won't take points off, but it probably would have been a better idea to look up the specific patients and doctors from their respective tables.